src/main/java/org/mdbenefits/app/cli/GoogleDriveClientImpl.java

```diff
@@ -33,7 +36,11 @@ public class GoogleDriveClientImpl implements GoogleDriveClient {
     private final String GOOGLE_CREDS = System.getenv("GOOGLE_DRIVE_CREDS");
     private final Drive service;
 
-    public GoogleDriveClientImpl() throws IOException, GeneralSecurityException {
+    private final String teamDriveId;
+
+    public GoogleDriveClientImpl(@Value("${transmission.google-drive-directory-id.team-drive:''}") String teamDriveId)
```

Comment:
For my own education: I'm curious what the argument to `@Value` does.

Reply:
@stabai the argument is a yaml field name that is located in the application.yaml (& profile overrides) file. It will pull from that field, if set in the yaml file, or default to whatever follows the colon, if supplied.